bindings/python/openshot.i

```diff
@@ -9,7 +9,7 @@
 //
 // SPDX-License-Identifier: LGPL-3.0-or-later
 
-%module openshot
+%module("threads"=1) openshot
```

Comment:
This allows all libopenshot methods to work async and not block the global Python lock